components/pattern-finder/src/main/java/org/datacleaner/beans/stringpattern/PatternFinderAnalyzer.java

```diff
@@ -118,13 +118,14 @@
     String[] predefinedTokenPatterns;
 
     @Configured(required = false, order = 12)
-    Character decimalSeparator = DecimalFormatSymbols.getInstance().getDecimalSeparator();
+    Character decimalSeparator = '.';
```

Comment:
Isn't it better to store the '.', ',' and '-' in constants, taking into account that you reuse them in the `TokenizerConfiguration` class

Reply:
Added constants in TokenConfiguration